airbyte-webapp/src/components/base/DropDown/DropDown.tsx

```diff
@@ -62,15 +62,13 @@ function DropDownInner<T = unknown>(
       zIndex: 9999,
     }),
   };
-
   return (
     <CustomSelect
       ref={ref}
       data-testid={props.name}
       $error={props.error}
       menuPlacement="auto"
       menuPosition="fixed"
-      menuPortalTarget={document.body}
```

Comment:
Looks like with the recent version of react-select this has been improved. I tried to removed this a while ago but there were some issues in the replication setting stream where the dropdown was under the overflow. This no longer seems to happen.